nodes/scheduling/nodes-scheduler-default.adoc

```diff
@@ -15,7 +15,12 @@ independent and exists as a standalone/pluggable solution. It does not modify
 the pod and just creates a binding for the pod that ties the pod to the
 particular node.
 
-A selection of predicates and priorities defines the policy for the scheduler. See xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default-modifying_nodes-scheduler-default[Modifying scheduler policy] for a list of predicates and priorities. 
+[IMPORTANT]
+====
+Configuring a scheduler policy is deprecated and might be removed in a future release. For more information on the Technology Preview alternative, see xref:../../nodes/scheduling/nodes-scheduler-profiles.adoc#nodes-scheduler-profiles[Scheduling pods using a scheduler profile].
```

Comment:
Same comment on the other PR, this will be removed

Reply:
@damemi but one question, as of today in 4.7 when customer tries to create scheduler policy file, we do not see any depecrated message, is this expected ?

Reply:
See https://github.com/openshift/openshift-docs/pull/28892#issuecomment-769074733, the operator does log a warning when setting `spec.policy`.

Reply:
@kasturinarra Let me know we are good to go with this PR, or if there's anything else we need to look into with the deprecation side. Thanks!

Reply:
@bergerhoffer we are good to go with this PR, i have tested and could see that scheduler operator logs throwing a warning which reads "W0202 16:43:03.106465       1 targetconfigcontroller.go:294] Setting .spec.policy is deprecated and will be removed eventually. Please use .spec.profile instead."